gno.land/genesis/genesis_params.toml

```diff
@@ -8,7 +8,7 @@
 
 ## gnovm
 ["gno.land/r/sys/params.vm"]
-  # TODO: chain_domain.string = "gno.land"
+  chain_domain.string = "gno.land"
```

Comment:
```suggestion
  chain.domain.string = "gno.land"
```

Reply:
could be applied to commented params below

Reply:
Oh, sorry. The category is "vm," not "chain." Let's use an underscore everywhere. Thanks.